packages/website/content/pages/faq.json

```diff
@@ -40,24 +40,34 @@
               },
               {
-                "id": "data-security",
-                "heading": "What happens if Web3.Storage is discontinued?",
-                "content": "One major advantage of using Web3.Storage is its use of IPFS, making it easy to move to an alternative storage provider. All of your data is accessible via IPFS, and you can simply export all of your CIDs and ask a pinning service to pin them without need to download or upload any data. You have this option even if you use Web3.Storage today and just want another redundant copy of your data persisted. And you don't need to change the URLs that your application is pointing to, since IPFS URLs are content addressable.<br><br>We don't expect to ever discontinue Web3.Storage, but if we need to, we'll give you 90 days' notice for you to make the easy switch."
+                "id": "pinning-api",
+                "heading": "How do I store data on web3.storage that is already available over the IPFS network without having to download and reupload it myself?",
+                "content": "Paid web3.storage plans give users access to our implementation of the <a href=https://ipfs.github.io/pinning-services-api-spec/' alt='Pinning Service API' target='_blank'>Pinning Service API</a>, which allows you to store data on web3.storage that is already available over the IPFS network."
+              },
+              {
+                "id": "service",
+                "heading": "What is the relationship between NFT.Storage and web3.storage?",
+                "content": "web3.storage is a platform that enables developers, applications, and users to interact with data independently of where it is stored, using decentralized protocols like <a href='https://ipfs.io/' alt='IPFS' target='_blank'>IPFS</a> and <a href='https://ucan.xyz/' alt='UCAN' target='_blank'>UCAN</a>. This includes an easy-to-use, performant storage service that makes data available on IPFS and persists it on the Filecoin decentralized storage network. NFT.Storage is a service whose mission is to perpetually store all off-chain NFT data as a public good (so for free, forever). It currently runs on the web3.storage platform today, but has future plans to provide its services in a fully decentralized manner (run by a DAO with operations and governance instrumented on smart contracts)."
```

Comment:
@dchoi27 we had touched this before, I think removing "but has future plans to provide its services in a fully decentralized manner (run by a DAO with operations and governance instrumented on smart contracts)."" has no downside and we wouldn't be talking on behalf of nft.storage for their future plans? -- not super critical to remove but wanted to give heads up again in case you think differently!

Reply:
i think we leave